src/components/atomic/poster.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+const PosterContainer = styled.div`
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	justify-content: center;
+`;
+
+const PosterStyle = styled.img<{ $borderRadius?: string }>`
+	${({ $borderRadius }) => $borderRadius && `border-radius: ${$borderRadius};`}
+	max-height: 100%;
+	max-width: 100%;
+	object-fit: contain;
```

Comment:
```suggestion
	max-height: 100%;
	object-fit: contain;
```

When setting the size of the poster, I would recommend only ever setting the width. In that case, you need to remove this to ensure it behaves consistently.

This also fixes the small posters at the bottom of the factoids disappearing on smaller screens.